### PR TITLE
community/opam: upgrade to 2.0.1

### PR DESCRIPTION
**Pull request by avsm**

### CODE REVIEW

**Comment by avsm** [comment]
@ncopa - would it be possible to get this into alpine 3.9 if the patch is ok? We've deprecated opam 1.2.2, so its package repository no longer regularly updates.

**Comment by ncopa** [comment]
Yes, i think we want this for v3.9, thanks!

**Comment by avsm** [comment]
Thanks @ncopa!

File: community/opam/APKBUILD [hunk 1]
```diff
@@ -1,18 +1,17 @@
 # Contributor: Michael Zuo <muh.muhten@gmail.com>
 # Maintainer: Anil Madhavapeddy <anil@recoil.org>
-# FIXME: lib-ext downloads dependencies; package them and use system-provided!
```
Comment (jirutka) [inline community/opam/APKBUILD]: Have you fixed this? Or opam 2.0.x does not download deps anymore?

**Pull request closed without merging**